src/fonduer/parser/parser.py

```diff
@@ -48,6 +47,9 @@ class Parser(UDFRunner):
     :param tabular: Whether to include tabular information in the parse.
     :param visual: Whether to include visual information in the parse.
         Requires PDFs for each input document.
+    :param vizlink: A custom visual linker that inherits
+        :class:`VisualLinker <fonduer.parser.visual_linker.VisualLinker>`.
+        Unless otherwise specified, :class:`VisualLinker` will be used. Default None.
```

Comment:
```suggestion
        Unless otherwise specified, :class:`VisualLinker` will be used.
```

This was a little confusing to say that something will be used, but defaults to None. I'd just say the above.